cloud

```diff
@@ -1675,6 +1681,7 @@ function execute_command() {
                           updated_publish \
                           submitters \
                           argv \
+                          schema \
```

Comment:
Let's keep the variable name consistent throughout the code, if we can. I think `schema_version` works well for that.

Reply:
okay